js/ndax.js

```diff
@@ -15,7 +15,11 @@ module.exports = class ndax extends Exchange {
             'id': 'ndax',
             'name': 'NDAX',
             'countries': [ 'CA' ], // Canada
-            'rateLimit': 1000,
+            // 1000 calls per hour
+            // 1000 / 60 = 16.666 calls per second
+            // 16.666 / 60 = 0.277 calls per second
+            // 1000ms / 0.277 = 3600
+            'rateLimit': 3600,
```

Comment:
I'm not sure if this is the correct value, have you tested it with lower values? Did it work with the previous rateLimit of 1000 ?

Reply:
I did not test it with the previous only with this one after seeing this in their policy docs: 

https://ndax.io/api-policy

<img width="643" alt="image" src="https://user-images.githubusercontent.com/87606888/162104113-4ae16d0d-3d48-496f-b8a5-9246deeb3f65.png">

Reply:
will test 1000 and see it its ok